content/install/server/auto_cancel.md

```diff
@@ -0,0 +1,37 @@
++++
+date = "2017-11-18T20:13:01+04:00"
+title = "Automatic Build Cancellation"
+url = "auto-cancel-builds"
+
+[menu.install]
+  identifier = "auto-cancel-builds"
+  parent = "install_server"
+  weight = 11
++++
+
+Drone supports automatically cancelling builds in either the running or pending state for the same branch.
+
+**NOTE**: This is an experimental feature that may cause issues if you have an existing installation
```

Comment:
Can you wrap this with `{{% alert error %}}` and `{{% /alert %}}` instead of `NOTE` please?

See https://github.com/drone/docs/blob/ae960fa88d0528af3d87f3194403ce6e95536b0d/content/usage/config/privileged.md